CONTRIBUTING.md

```diff
@@ -129,88 +123,61 @@ You can get these values by registering a new integration on [Cisco Webex for De
-   - DEBUG - if your test source includes the debug package set this to the appropriate string to enable debug output
-     For exampe if you want to run only the plugin-messages test, and see the package specific logging, your command line would be:
-     > `WEBEX_LOG_LEVEL=log DEBUG=messages yarn run test --packages @webex/plugin-messages --node`
-
-5. Repeat steps 2-3 until the tests pass.
-   > If you use VS Code, we've created a configuration to utilize the built-in debugger
-   >
-   > - Set breakpoints within the package you're working on
-   > - Select the `Test package` configuration
-   > - Enter the package you'd like to test (i.e. `MYPACKAGE`)
-   >   - _The configuration already prepends `@webex/` for you unlike the cli command, so just `plugin-teams` is fine_
-   > - Add any _optional_ flags (i.e. `--node`)
-   >   - _If you don't want to add any flags, just add a space (current workaround)_
-6. Run `yarn run srcdist` to restore the package.jsons to avoid committing those changes.
-
-You can use the `--unit`, `--integration`, `--automation`, and `--documentation` switches to control what types of tests you run and `--node` and `--browser` to control which environments your tests run in.
-
-The `--packages` flags will allow you to test multiple packages in one command instead of separate commands for each package `--packages @webex/plugin-meetings @webex/plugin-rooms @webex/plugin-teams`. Packages are still tested synchronously to allow for proper output to the terminal.
-
-`--browser --karma-debug` will run the browser tests with `{singleRun: false}`, thus allowing automatic rerunning every time you save a file (though, karma does eventually get confused and you need to interrupt and restart the command).
+`test:browser --karma-debug` will run the browser tests with `{singleRun: false}`, thus allowing automatic rerunning every time you save a file (though, karma does eventually get confused and you need to interrupt and restart the command).
 
 You can use the `--browsers` _(not to be confused with the `--browser` tag)_ allows you to specify which browsers you want to run locally. This is restricted to what browsers are installed and available to you on your OS.
-The default browsers that launch are _Headless_ version of Firefox and Chrome, so `--browsers Chrome Edge` will only launch a normal version of Chrome along with Edge. If you add `defaults` to the browsers flag, it will also launch `ChromeHeadless` and `FirefoxHeadless` along with other browsers you've specified. All browsers include flags to enable WebRTC features and permissions.
+The default browsers that launch are _Headless_ versions of Firefox and Chrome, so `--browsers Chrome Edge` will only launch a normal version of Chrome along with Edge. If you add `defaults` to the browser flag, it will also launch `ChromeHeadless` and `FirefoxHeadless` along with other browsers you've specified. All browsers include flags to enable WebRTC features and permissions.
```

Comment:
Can we also specify a note that Chrome and Firefox must be installed in their machine?

Reply:
Makes sense @mkesavan13 . I think we can also add the step for setting up the environment variable.